zenoh/src/lib.rs

```diff
@@ -289,29 +285,33 @@ pub mod publisher {
             PublicationBuilder, PublicationBuilderDelete, PublicationBuilderPut, PublisherBuilder,
             PublisherDeleteBuilder, PublisherPutBuilder,
         },
-        publisher::{Priority, Publisher, PublisherUndeclaration},
+        publisher::{Publisher, PublisherUndeclaration},
     };
 }
 
-/// Query primitives
+/// Get operation primitives
+pub mod querier {
+    // Later the `Querier` with `get`` operation will be added here, in addition to `Session::get`,
```

Comment:
If the `Querier` is not ready, I don't think we should have a module for now.
Also, the same way than `put` relative types are in the module `publisher`, `query` module itself should be renamed `queryier`, shouldn't it?

Reply:
Ok, but I still think we should just rename the `query` module to be consistent with `publisher` module.

Reply:
`query` module logically now similar to `sample`. I.e. 
`publisher` is a server, `subscriber` is a client, `sample::Sample`, etc is data which circulates between them
`queryable` is a server, `querier` is a client, `query::Query`, `query::Reply` is data

Reply:
`query` is not similar to `sample`, btw `Query` is a member of `queryable`, not `query`.
`query` module is here to provide the types specifically used by `Session::get`, and when `get` will be abstracted into a `Querier` object, `query` module should be renamed to `querier` to stay consistent with `publisher`.